app/views/admin/newsletters/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= back_link_to %>
+
+<h2><%= t("admin.newsletters.show.title") %></h2>
+
+<%- recipients_count = @newsletter.list_of_recipients.count %>
```

Comment:
I just learned about <%- (https://puppet.com/docs/puppet/4.9/lang_template_erb.html#space-trimming-1) but I wonder if it was intentional... and if so ... should we use it as an standard good practice?

Reply:
Totally! I try to use it as a _personal convention_, but we can discuss if is worth it to use it as our standard 🙂